### PR TITLE
Adding LPaaS 2.0 Content

### PR DESCRIPTION
**Pull request by saifulislamdev**
Older PR was attempting to merge to `main`.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,7 +2,9 @@ node_modules/
 
 # Gatsby
 .cache/
-public/
+
+# Gatsby + production
+public/
```
Comment (aannyyaa) [inline .gitignore:7]: Why separate this from the Gatsby section?

File: src/components/hero.jsx [hunk 2]
```diff
@@ -12,9 +13,14 @@ function Hero({ buttons, hero }) {
             <h1 className="usa-hero__heading">{hero.heading}</h1>
           )}
           {hero.text && <p className="usa-prose">{hero.text}</p>}
-          {buttons.callToAction && (
-            <Button className="usa-button--inverse">
-              {buttons.callToAction}
+          {hero.buttonLink && hero.buttonText && (
```
Comment (aannyyaa) [inline src/components/hero.jsx:16]: Good update on the buttons!

File: src/markdown-pages/landing-ar.md [hunk 3]
```diff
@@ -4,33 +4,35 @@ templateKey: landing-page
 slug: home
 pageTitle: ar Project | Home
 hero:
-  image: ar hero-img.jpg
+  image: hero-img.jpg
```
Comment (aannyyaa) [inline src/markdown-pages/landing-ar.md]: We can actually delete hero.image and hero.imageDescription from all markdown files and from the GraphQL query. The hero background image is set in the theme

**Review by aannyyaa — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (saifulislamdev) [reply .gitignore:7]: I wanted to add further clarification for what it was for.

File: src/components/hero.jsx (hunk 2 above)
Comment (saifulislamdev) [reply src/components/hero.jsx:16]: Thanks :)

File: src/markdown-pages/landing-ar.md (hunk 3 above)
Comment (saifulislamdev) [reply src/markdown-pages/landing-ar.md]: Done. Thanks for reminding me to change the GraphQL query, too. I'm assuming there is no`alt` text?

**Pull request merged**